templates/addons/aurora/cf.yml

```diff
@@ -31,6 +31,34 @@ Mappings:
       {{end -}}
     {{end}}
 Resources:
+  {{logicalIDSafe .ClusterName}}AuroraKMSCMK:
+    Type: 'AWS::KMS::Key'
+    DeletionPolicy: Retain
+    Properties:
+      KeyPolicy:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
+            Action: 'kms:*'
+            Resource: '*'
```

Comment:
Can we limit these resources to just the Aurora DB?

Reply:
This statement is to prevent the key from being unmanageable in the future ([doc]( https://docs.aws.amazon.com/kms/latest/developerguide/key-policies.html#key-policy-default)). KMS guards against this sort of unmanageable situation and the resource will fail to create.